src/client/datascience/notebook/helpers/helpers.ts

```diff
@@ -663,36 +663,46 @@ function convertOutputMimeToJupyterOutput(mime: string, value: Uint8Array) {
     if (!value) {
         return '';
     }
-    const stringValue = Buffer.from(value as Uint8Array).toString('utf8');
-    if (mime === CellOutputMimeTypes.error) {
-        traceInfo(`Concerting ${mime} from ${stringValue}`);
-        return JSON.parse(stringValue);
-    } else if (mime.startsWith('text/') || textMimeTypes.includes(mime)) {
-        return stringValue;
-    } else if (mime.startsWith('image/')) {
-        // Images in Jupyter are stored in base64 encoded format.
-        // VS Code expects bytes when rendering images.
-        return Buffer.from(stringValue, 'base64');
-    } else if (mime.toLowerCase().includes('json')) {
-        return JSON.parse(stringValue);
-    } else {
-        return stringValue;
+    try {
```

Comment:
Added try catch to prevent crashing everything.
This way we can at least open the ipynb file.